src/main/scala/vsys/contract/Contract.scala

```diff
@@ -1,34 +1,88 @@
 package vsys.contract
 
+import com.google.common.primitives.Ints
 import com.wavesplatform.state2.ByteStr
-import scorex.serialization.{BytesSerializable, Deser}
+import com.wavesplatform.utils.base58Length
+import play.api.libs.json.{JsObject, Json}
+import scorex.crypto.encode.Base58
+import scorex.serialization.Deser
 import scorex.transaction.ValidationError
+import scorex.transaction.ValidationError.InvalidContract
+
+import scala.util.Success
 
 sealed trait Contract {
 
-  lazy val stringRepr: String = Contract.Prefix + ":" + name
-  lazy val bytes: ByteStr = ByteStr(BytesSerializable.arrayWithSize(name.getBytes("UTF-8")) ++
-    (if (enabled) Array(1: Byte) else Array(0: Byte)) ++
-    content.getBytes("UTF-8"))
+  lazy val stringRepr: String = Contract.Prefix + Base58.encode(languageCode) + ":" + Base58.encode(languageVersion)
```

Comment:
why ":" appeared after `languageCode` ?

Reply:
To separate langCode and langVersion. Is this needless?